contributing/release-schedule.md

```diff
@@ -1,84 +1,84 @@
 
-### Cherry pick criteria
+Important: Publishers using the **lts** release channel should not use newly introduced features. Because of the four-week cycle, the **lts** release may be as much as six weeks behind the `HEAD` of [`ampproject/amphtml`](https://github.com/ampproject/amphtml). See the section on [determining if your change is in a release](#Determining-if-your-change-is-in-a-release).
 
-**The bar for getting a cherry pick into canary or production is very high** because our goal is to produce high quality launches on a predictable schedule.
 
-**Keep in mind that performing a cherry pick requires a significant amount of work from you and the onduty person** and they can take a long time to process.
+## Determining if your change is in a release
 
-- In general only fixes for [P0 issues](https://github.com/ampproject/amphtml/blob/master/contributing/issue-priorities.md) (causing "an outage or a critical production issue") may be cherry picked. P0 issues are those that:
-  - cause privacy or security issues
-  - cause user data loss
-  - break existing AMP web pages in a significant way
-  - or would otherwise cause a significant harm to AMP's reputation if left unresolved
-- Regressions found in the canary that are not P0 _may_ be approved if they can be resolved with a rollback. Fixes other than rollbacks--no matter how simple they may seem--will not be approved because these can cause cascading problems and delay the release of canary to production for everyone.
+[_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) are used to track the status of current and past releases (from the initial cut to **experimantal**/**beta** testing to **stable**). Announcements about releases are made on the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
 
-### Process for requesting a cherry pick
+You can determine what changes are in a given build using one of the following:
 
-Use the following process to request a cherry pick if you have a change that you believe meets the [cherry pick criteria](#cherry-pick-criteria).
+- The [_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for each release build will include a link to the specific [release page](https://github.com/ampproject/amphtml/releases) listing the PRs in that release.
+  - The most recent _Type: Release_ issue with _(Production)_ in the title tracks the build currently in the **stable** release.
+  - The _Type: Release_ issue with _(Canary)_ in the title tracks the build currently in the **experimental**/**beta** releases.
```

Comment:
Pretty sure these two sentences are not accurate. We [do not use](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Type%3A+Release%22+%22release+tracking+issue%22+) suffixes in the title to tell release issues apart. Is something changing? (Or am I missing something?)

If I'm right, I suggest just deleting the above two sentences.

Reply:
I'm not sure (this was a carry-over from the OG docs). I'll remove these lines since they seem to be deprecated.